infra/base-images/base-runner/Dockerfile

```diff
@@ -115,6 +115,7 @@ COPY bad_build_check \
     reproduce \
     run_fuzzer \
     parse_options.py \
+    profraw-update.py \
```

Comment:
nit: Maybe use underscores in name so it is a valid python module.

Reply:
done